website/src/en/playground.html

```diff
@@ -24,7 +24,7 @@
   <link rel="stylesheet" crossorigin  href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.26.0/theme/neat.css">
```

Comment:
This is going to break the playground for existing users if we merge this. I'm fine publishing the website even if it's super barebones.